### PR TITLE
Fixes #328

### LINKED ISSUE #328: comment-format check-uppercase
There is "check-lowercase" but there is no "check-uppercase"

### PR DESCRIPTION
**Pull request by NN---**

### CODE REVIEW

File: src/rules/commentFormatRule.ts [hunk 1]
```diff
@@ -119,4 +127,21 @@ class CommentWalker extends Lint.RuleWalker {
         }
     }
 
+    private startsWithUppercase(commentText: string): boolean {
```
Comment (adidahiya) [inline src/rules/commentFormatRule.ts]: This shares a great deal of logic with `startsWithLowercase`. The two methods should be refactored for DRY.

**Comment by adidahiya** [comment]
The build is failing because `tslint` failed on `commentFormatRuleTests.ts`. Please run ensure the project builds (simply via `grunt`) on your local machine.

File: src/rules/commentFormatRule.ts (hunk 1 above)
Comment (NN---) [inline src/rules/commentFormatRule.ts]: OK

**Comment by NN---** [comment]
Fixed.

**Comment by gscshoyru** [comment]
Please also update the readme to include this new option.

**Comment by NN---** [comment]
No problems

**Pull request merged**